nautobot/dcim/tests/test_api.py

```diff
@@ -307,7 +323,14 @@ def setUpTestData(cls):
 
     def get_deletable_object_pks(self):
         """To get the correct bulk-delete object count, make sure we avoid a cascade deletion."""
-        return [loctype.pk for loctype in list(LocationType.objects.filter(children__isnull=True))[:3]]
+        lt1 = LocationType.objects.create(name="Country")
+        lt2 = LocationType.objects.create(name="State")
+        lt3 = LocationType.objects.create(name="County")
+        return [lt1.pk, lt2.pk, lt3.pk]
+
+    def get_deletable_object(self):
+        """To get the correct delete object count, make sure we avoid a cascade deletion."""
+        return LocationType.objects.create(name="delete this")
```

Comment:
These can be removed after you merge in my changes from https://github.com/nautobot/nautobot/pull/2579
However you'll need to create 3 `LocationType` instances in your `setUpTestData` since all of the `LocationTypes` created by the factories are being used